features/support/api/http.js

```diff
@@ -607,13 +609,7 @@ class HttpApi {
   }
 
   getStats (dates) {
-    const options = {
-      url: this.apiPath('_getStats'),
-      method: 'POST',
-      body: dates
-    };
-
-    return this.callApi(options);
+    return this.callApi(this._getRequest(null, null, 'server', 'getStats', {body: dates}));
```

Comment:
Why are the `dates` arguments enclosed in a body?

Reply:
It's because of actual `_getRequest` implementation, that injects `args.body` parameters into query string: 
https://github.com/kuzzleio/kuzzle/blob/1686a257384e12798998071ebeabf0a04704e4e0/features/support/api/http.js#L82

(NB: It's a temporary fix, as I'm working on totally refactoring feature tests, and use javascript SDK instead of custom implementation to connect to kuzzle)

Reply:
oh, right, I forgot about that